### ISSUE TITLE
how to run 

### ISSUE THREAD

**Issue by sri6027** [KenanNiu/vtk3DVolumereconstruction-python#1]
Hi, how to run this file as i am new to python . i have installed python and VTK also and trying to ru from the command prompt like "PYTHON volume3DReconstruction.py" IS this correct ??

**Comment by KenanNiu** [comment]
Yes. You can run in this way. Be aware of the image folder directory in the code. line 55. It should be compatible with your image files.

**Comment by sri6027** [comment]
thanks for the replying 
![cmd](https://user-images.githubusercontent.com/36832951/115524858-2e4ffa00-a2ac-11eb-9457-a4a32788b6c4.png)
![dcmfiles](https://user-images.githubusercontent.com/36832951/115524862-2f812700-a2ac-11eb-848a-efff1f085567.PNG)
i have attached the error screen shot and input data screen shot can u please check what is the error

**Comment by KenanNiu** [comment]
hi, 
I saw you put dicom for loading, while this example is made for a series of png images. For dicom, you need to modify the code where they do image reading....

`png_reader = vtk.vtkPNGReader()
png_reader.SetFileName(join(imagefolderpath, imagepathlist[0]))
png_reader.Update()
print(png_reader.GetOutput().GetDimensions())`

**Comment by sri6027** [comment]
what to change in these lines .

**Comment by KenanNiu** [comment]
import vtk
from vtk.util import numpy_support
import numpy

PathDicom = "./dir_with_dicom_files/"
reader = vtk.vtkDICOMImageReader()
reader.SetDirectoryName(PathDicom)
reader.Update()
